Core/CoreTests/Contexts/ContextColorTests.swift

```diff
@@ -22,20 +22,24 @@ import Foundation
 import XCTest
 
 class ContextColorTests: CoreTestCase {
+
     func testSave() {
-        let response = APICustomColors(custom_colors: ["course_1": "#000", "group_1": "#fff"])
+        let blackColor = UIColor(hexString: "#000")!
+        let whiteColor = UIColor(hexString: "#fff")!
+        let response = APICustomColors(custom_colors: ["course_1": blackColor.hexString, "group_1": whiteColor.hexString])
         let result = ContextColor.save(response, in: databaseClient)
         XCTAssertEqual(result.count, 2)
+
         let course = result.first { $0.canvasContextID == "course_1" }!
-        XCTAssertEqual(course.color.hexString, "#000000")
-        let group = result.first { $0.canvasContextID == "group_1" }!
-        XCTAssertEqual(group.color.hexString, "#ffffff")
-    }
+        XCTAssertEqual(
+            course.color.hexString,
+            CourseColorsInteractorLive().courseColorFromAPIColor(blackColor.hexString).hexString
```

Comment:
1. Changed to the other method.
2. I see your point but I wouldn't make `CoreTestCase` more heavier. Instead a proper DI system should be introduced because in most of these test we are not interested in the actual color code just that if it came through the color interactor or not.
3. I changed the color codes.